test_projects/android/bash/generate_apks.sh

```diff
@@ -1,5 +1,7 @@
 #!/usr/bin/env bash
 
+echo "DEPRECATED!!!"
```

Comment:
Can we add more descriptive message? Like why it was deprecated / what's the new way of generating it etc.